tests/contrib/anthropic/test_anthropic_async.py

```diff
@@ -0,0 +1,142 @@
+import pytest
```

Comment:
We can also move the async tests to the same `test_anthropic.py` file

Reply:
Also, is it possible to reuse the same snapshot and cassette files for both sync and async tests? I would prefer to minimize the number of static test data files we keep.

Reply:
done